src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,10 @@
+import Calculator from '../components/Calculator';
+
+const CalculatorPage = () => (
+  <div className="calculator-div">
+    <h1>Let´s do some math!</h1>
+    <Calculator />
+  </div>
```

Comment:
- [ ] Nice design 👏 . lets adjust this text to match the aligmnet of the main header, just as it is on the wireframe given, to have a perfect replica 👍 

###  Templet:
![image](https://github.com/EvansSnave/math-magician-2/assets/50990925/1d7cde1b-3986-4262-bf42-34694d9e2fa5)

### View:
![image](https://github.com/EvansSnave/math-magician-2/assets/50990925/8b7e127f-8f36-471f-9c7b-deeb218f1858)